README.md

```diff
@@ -4,6 +4,25 @@ git clone https://github.com/theforeman/smart-proxy
 ```
 Configure smart proxy
 
+
+In smart-proxy directory 
+```shell
+mkdir smart-proxy/logs
+```
+Create two new files in smart-proxy/config/settings.d --> First one named `dynflow.yml` and second one `remote_execution_ssh.yml` and add lines  to both of them 
```

Comment:
It would be better to copy the example config files from the plugins instead of just enabling them